_posts/2020-06-30-not-only-the-code-benefits-from-the-open-source-community.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Not Only The Code Benefits From The Open Source Community
+date: 2020-06-30  9:08:11
+author: Sergei Petrosian
+
+tags:
+- foreman
+- katello
+- documentation
+---
+
+The upstream versions of the Foreman and Katello projects become among other projects like Atix's Orcarhino, the Red Hat Satellite of tomorrow. The discussions and contributions that take place in the vibrant upstream community shape the Red Hat Satellite code base. The code base for Satellite begins upstream and moves downstream. Until now, the Satellite documentation did not follow the same journey.
+
+<!--more-->
+
+While a team of Red Hat technical writers work on Satellite documentation full time, Foreman and Katello documentation is written by the engineering team and community members who suggest updates and corrections. Because of this, there has been a huge disparity between the types of documentation that is produced upstream and downstream.
+
+For example, Foreman documentation focuses on features and all possible functions of those features. This documentation is closer in spirit to man pages that describe all things that you can do with a single command. While thorough and accurate, the documentation does not focus on a particular user scenario so might prove hard to use to accomplish certain tasks. For example, in the Foreman manual, section [4.1.2 Roles and Permissions](https://theforeman.org/manuals/2.0/index.html#4.1.2RolesandPermissions) provides an overview of users and roles in Foreman but does not delve into the issue of why this information is relevant or walk the user through doing it. A similar chapter that is written for the Satellite guide by a technical writer, [5. Managing Users and Roles](http://docs.theforeman.org/guides/build/doc-Administering_Red_Hat_Satellite/index-foreman.html#chap-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Users_and_Roles) provides a set of concepts and procedures that define both the what and then how users can accomplish this.
+
+Apart from this main difference in the idea of how to document the product, it is worth mentioning other differences between the traditional and the new documentation:
```

Comment:
For me the placement of this numbered list doesn't make any sense. I think it would be best to come at the end .